core/domain/change_domain.py

```diff
@@ -119,7 +117,7 @@ class BaseChange:
     # This is a list of common commands which is valid for all subclasses.
     # This should not be overriden by subclasses.
     COMMON_ALLOWED_COMMANDS = [{
-        'name': base_models.VersionedModel.CMD_DELETE_COMMIT,
+        'name': feconf.CMD_DELETE_COMMIT,
```

Comment:
Why does this need to be defined in feconf? I would suggest moving it to `change_domain` as is a change command (though I'm not sure whether we can import domain files in the model layer).

Reply:
That attribute was already in `feconf` I just used it here because I had to remove the dependency on `base_models` as that was the suggested fix to resolve errors while pushing - https://github.com/oppia/oppia/pull/13772#discussion_r701800651

Reply:
> @subhash686 Hey, please do not resolve comments yourself, always let the reviewer do that. If you've fixed some comment reply to it with "Done".

Got it. Unresolved now. Thanks!